### PR TITLE
Release/0.1.24 monorepo

### PR DESCRIPTION
**Pull request by andyh565**

### CODE REVIEW

File: packages/connectors/hull-sql/assets/index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<html>
+<head>
+  <title>Segment Ship</title>
```
Comment (timliuhull) [inline packages/connectors/hull-sql/assets/index.html:3]: Was this in the sql connector too?

File: webpack/webpack.config.js [hunk 2]
```diff
@@ -58,6 +58,10 @@ const buildConfig = ({ assets, files, destination, mode = "production" }) => ({
 
   module: {
     rules: [
+      {
+        test: /\.css$/,
+        use: [{ loader: "style-loader" }, { loader: "css-loader" }]
+      },
```
Comment (timliuhull) [inline webpack/webpack.config.js:64]: a bit out of the way for this change, was this intended?

File: packages/connectors/hull-sql/assets/index.html (hunk 1 above)
Comment (andyh565) [reply packages/connectors/hull-sql/assets/index.html:3]: ah yeah I didn't catch that. It was in the original sql connector

File: webpack/webpack.config.js (hunk 2 above)
Comment (andyh565) [reply webpack/webpack.config.js:64]: I'll double check this. It may have been left over from when I was trying to get webpack to work

**Comment by unity** [comment]
Since we're gonna have to QA this - would it be valuable to move the import strategy to the Kakfa adapter at the same time ?

**Comment by andyh565** [comment]
> Since we're gonna have to QA this - would it be valuable to move the import strategy to the Kakfa adapter at the same time ?

That makes sense to me. @timliuhull what do you think?

**Comment by unity** [comment]
It would make it possible to support the session ID capture and other things we don't support via the import API without needing platform-side changes.

**Comment by timliuhull** [comment]
> > Since we're gonna have to QA this - would it be valuable to move the import strategy to the Kakfa adapter at the same time ?
> 
> That makes sense to me. @timliuhull what do you think?

I think the Import Api needs to be able to handle session_id etc... Unless we think the Import Api is not the way we want to do bulk imports in the future.  

Kafka adapter was a replacement for the firehose, not imports.  I don't know the details, but I'd think a bulk import endpoint would always be useful, especially in cases where we're pulling stuff directly from a database like this and the volume could be pretty large.

And just as a general instinct, I'd try not to put too many things together and make an already big story bigger.

**Comment by andyh565** [comment]
> > > Since we're gonna have to QA this - would it be valuable to move the import strategy to the Kakfa adapter at the same time ?
> > 
> > 
> > That makes sense to me. @timliuhull what do you think?
> 
> I think the Import Api needs to be able to handle session_id etc... Unless we think the Import Api is not the way we want to do bulk imports in the future.
> 
> Kafka adapter was a replacement for the firehose, not imports. I don't know the details, but I'd think a bulk import endpoint would always be useful, especially in cases where we're pulling stuff directly from a database like this and the volume could be pretty large.
> 
> And just as a general instinct, I'd try not to put too many things together and make an already big story bigger.

Yeah I think this original story is closed and we should be able to merge it and anything else will need to be additional stories. There's no risk in merging this as is as no one will be using it right now anyway

File: webpack/webpack.config.js (hunk 2 above)
Comment (andyh565) [reply webpack/webpack.config.js:64]: @timliuhull yes it's needed for the package codemirror

**Pull request merged**